docker-tools/Dockerfile-from-travis

```diff
@@ -1,16 +1,21 @@
 #/* vim: set filetype=dockerfile : */
-FROM travisci/ci-garnet:packer-1512502276-986baf0
+FROM travisci/ci-garnet:packer-1515445631-7dfb2e1
 
 ADD . /tmp/bokeh
 RUN chown -R travis /tmp/bokeh
 
 USER travis
 
 ENV NO_GIT_FETCH=1
-ENV PATH="/home/travis/miniconda/bin:${PATH}"
-ENV CONDA_REQS="conda=4.4.11 conda-env=2.6.0 conda-build=3.0.27"
+ENV PATH="/home/travis/miniconda/envs/testenv/bin/:/home/travis/miniconda/bin/:${PATH}"
+ENV CONDA_REQS="conda=4.6.2 conda-build=3.17.8"
```

Comment:
we should see if we can bump to 4.7 versions. The latest condas use a different compression scheme that is substantially faster

Reply:
I got this from the travis settings, and do not dare to touch them. Feel free though!

Reply:
OK, I am going to try and rev those values on TravisCI after the upcoming release